.infrastructure/60_cdn.tf

```diff
@@ -0,0 +1,108 @@
+resource "aws_cloudfront_origin_access_identity" "main" {
+  comment = "Identity to access S3 bucket."
+}
+
+## Static website CDN
+resource "aws_cloudfront_distribution" "website" {
```

Comment:
in future we will have multiple websites. did you already think on how we should handle the resources on this use case?

because in that scenario, I don't think `website` should be a meaningful name

Reply:
Now consider only the use case of a single site. In the future, we may use the same cloudfront instance with multiple origins (i.e. one S3 bucket for each site) or create one cloudfront instance for each different origin (site). Change the name from "website" to "website_send" ?

Reply:
ok, since it's not clear right now how we are going to handle the scenario with multiple websites, then I think it's ok to keep `website` as resource name, as long it is possible to update it in future, and change the name in future. what do you think @maurodandrea?

Reply:
Yes i agree. In future it is possible update the name.